integration/pod_userns_linux_test.go

```diff
@@ -164,3 +303,25 @@ func TestPodUserNS(t *testing.T) {
 		})
 	}
 }
+
+func supportsRuncIDMap() error {
+	var r runc.Runc
+	features, err := r.Features(context.Background())
+	if err != nil {
+		// If the features command is not implemented, then runc is too old.
+		return fmt.Errorf("features command failed: %w", err)
+	}
+	if features == nil {
+		return errors.New("features is nil")
+	}
+
+	if features.Linux.MountExtensions == nil || features.Linux.MountExtensions.IDMap == nil {
+		return errors.New("missing entry in features command")
```

Comment:
```suggestion
		return errors.New("missing `mountExtensions.idmap` entry in `features` command")
```

Reply:
Fixed, thanks!

Reply:
Will push after the current CI run finishes, to see the results (it takes a lot of time, don't want to reset it now :))